apis/nodecore/v1alpha1/allocation_types.go

```diff
@@ -23,11 +23,13 @@ type Status string
 
 // Status is the status of the allocation.
 const (
-	Active   Status = "Active"
-	Reserved Status = "Reserved"
-	Released Status = "Released"
-	Inactive Status = "Inactive"
-	Error    Status = "Error"
+	Active           Status = "Active"
+	Provisioning     Status = "Provisioning"
```

Comment:
What's the difference between `Provisioning` and `ResourceCreation`?
Why do we need them both?

Reply:
The Allocation usually start from the Inactive state, it then instantly switches to the Peering phase, which creates Liqo ForeignCluster and start the peering. This phase can be long, since the Allocation Controller is triggered even upon ForeignCluster changes, I though it would have been usefull to have an intermediate phase called Provisioning which is used to check the peering status. Once the Provisioning phase has been completed (the Liqo peering has been established), the Allocation can now create the resources related to the service (ResourceCreation).